pinc/Activity.inc

```diff
@@ -360,13 +396,13 @@ function get_user_scores($username, $criteria)
     $user_obj = $username == User::current_username() ? User::load_current() : new User($username);
 
     $user_scores = [];
-    foreach ($criteria as $criterion_code => $criterion_descr) {
+    foreach ($criteria as $criterion_code => $_) {
```

Comment:
Would this be clearer as
```suggestion
    foreach (array_keys($criteria) as $criterion_code) {
```
?

Reply:
I am not super opinionated on those so included the change as it made sense.